core/contractsdk/go/exec/contract_context.go

```diff
@@ -18,7 +18,7 @@ const (
 	methodGetCallArgs       = "GetCallArgs"
 	methodTransfer          = "Transfer"
 	methodContractCall      = "ContractCall"
-	methodCrossContractCall = "CrossContractCall"
+	methodCrossContractCall = "CrossContractQuery"
```

Comment:
ContractCall could be query or invoke， do we need keep it consistent by change 'methodCrossContractCall' to 'methodCrossContractQuery'?

Reply:
Done.